src/Plugin.php

```diff
@@ -99,9 +99,14 @@ public static function init() {
     add_action('woocommerce_process_product_meta', __NAMESPACE__ . '\WooCommerce::woocommerce_process_product_meta');
     add_action('woocommerce_save_product_variation', __NAMESPACE__ . '\WooCommerce::woocommerce_save_product_variation', 10, 2);
 
-// Adds woocommerce specific settings.
+    // Adds woocommerce specific settings.
     add_filter('woocommerce_get_settings_shop_standards', __NAMESPACE__ . '\WooCommerce::woocommerce_get_settings_shop_standards');
 
+    // Assigns sale category conditionally on product update.
```

Comment:
The lines are no longer removed from Admin since the rebase:
https://github.com/netzstrategen/wordpress-shop-standards/blob/a8c648b4c65c44f754dedd527950ef6a95b8bf6a/src/Admin.php#L29-L32